components/eamxx/docs/developer/dev_testing/full_model_testing.md

```diff
@@ -0,0 +1,199 @@
+# Full-model Testing
+
+Full model system testing of EAMxx is done via CIME test cases
+(much like the rest of E3SM).
+
+We offer a number of test suites, including:
+
+- `e3sm_scream_v0`: Test the full set of V0 (pre-C++) tests
+- `e3sm_scream_v1`: Test the full set of V1 (C++) tests
+- `e3sm_scream_v1_at`: A smaller and quicker set of tests for autotesting (AT)
```

Comment:
Again, we don't use this, so let's not mention it. It is available for ppl to run, but still.